fw/t/unit/Makefile

```diff
@@ -29,15 +29,15 @@ obj-m += tfw_test.o
 t-srcs = $(wildcard $(obj)/*.c)
 t-objs = $(patsubst %.c, %.o, $(t-srcs))
 ifdef AVX2
-	t-objs += ../../str_avx2.o
+	t-objs += str_avx2.o
 endif
 
 tfw_test-objs = $(subst $(src)/,,$(t-objs))
 
 tfw_test-y +=	\
-	../../http_parser.o	\
-	../../http_sess.o	\
-	../../str.o		\
-	../../ss_skb.o		\
-	../../msg.o		\
-	../../http_msg.o
+	http_parser.o	\
+	http_sess.o	\
+	str.o		\
+	ss_skb.o	\
+	msg.o		\
+	http_msg.o
```

Comment:
The fix looks quite subtle, so I think at least a comment in the Makefile is required or we'll be confused by the files with just includes.

Reply:
Added a description into this Makefile.
During the process I realized that there is no need to add anything to `tfw_test-y`, as `.c` symlinks are already being handled by `$(wildcard $(obj)/*.c)` some lines above. So that unnecessary part was removed.